Tests/Helpers/Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse+Test.swift

```diff
@@ -1,9 +1,22 @@
 import XpringKit
 
+// TODO(keefertaylor): Refactor Org_Xrpl_Rpc_V1_AccountAddress to be a static test helper object to avoid repetition.
 extension Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse {
   static let testTransactionHistoryResponse = Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse.with {
     $0.account = Org_Xrpl_Rpc_V1_AccountAddress.with {
       $0.address = .testAddress
     }
+
+    $0.transactions = [
```

Comment:
What does the `$0` here mean?

Reply:
yeah I'd love to know this too

Reply:
This is kinda complicated. 

this method is defined as:
```
    public static func with(_ populator: (inout Self) throws -> ()) rethrows -> Self
````

What this says is: 
- `with` is a static method  (`public static func with(`)
- which takes a closure (you could also call this an anonymous function or lambda in other languages) (`_ populator: (inout Self) throws -> ())
- This block takes an inout parameter of type self
- And the whole function returns an instance of `Self` (`-> Self`)

So, we could write something like this (long form):
```
Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse.with((populator: Self) in 
  populator.transactions = <...>
)
````

Having the last parameter of a method be a closure is pretty common in swift, so we actually have special syntax when we write outside of the method ("trailing closure syntax"):

```
Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse.with() { (populator: Self) in 
  populator.transactions = <...>
}
```

Trailing closure syntax also lets us drop the `()` for brevity if there are no other parameters. And we can drop the type annotation in the parameter to the closure:
```
Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse.with { populator in 
  populator.transactions = <...>
}
```

Swift also lets us drop the names of parameters and refer to them as `$<param number` in closures, so we can take this one step further:

```
Org_Xrpl_Rpc_V1_GetAccountTransactionHistoryResponse.with { 
  $0.transactions = <...>
}
```

(This is in fact the recommended syntax from the Swift guide: https://github.com/apple/swift-protobuf#quick-start)

Reply:
In general, Swift has some gnarly closure syntax if you don't know what you're looking at. The definitive reference is: 
https://fuckingclosuresyntax.com/

You might also find this guide useful: https://www.hackingwithswift.com/example-code/language/what-is-trailing-closure-syntax